docs/nx/logger/winston.md

```diff
@@ -0,0 +1,69 @@
+<summary>Add Winston to your project</summary>
+Generates a new Nx library which contains a Winston logger instance and associated config.
+
+## Prerequisites
+
+An existing Stacks workspace.
+
+## Usage
+
+```bash
+nx @ensono-stacks/logger:winston
+```
+
+### Command line arguments
+
+The following command line arguments are available:
+
+| Option              | Description                                                      | Type    | Accepted Values | Default | Required? | Available in interactive prompt? |
+|---------------------|------------------------------------------------------------------|---------|-----------------|---------|-----------|----------------------------------|
+| --name              | Name of the generated library                                    | string  |                 | N/A     | Yes       | Yes                              |
+| --tags              | Add tags to the project (used for linting)                       | string  |                 | N/A     |           |                                  |
+| --directory         | Directory where the project is placed (within Nx libs directory) | string  |                 | N/A     |           |                                  |
+| --skipFormat        | Skip formatting files                                            | boolean | true/false      | false   |           |                                  |
+| --logLevelType      | The type of log levels that will be used                         | enum    | cli/syslog/npm  | npm     |           | Yes                              |
+| --consoleLogs       | Output logs to the console                                       | boolean | true/false      | false   |           |                                  |
+| --fileTransportPath | File path used for logs transport                                | string  |                 | N/A     |           |                                  |
+| --httpTransport     | Add a http transport                                             | boolean | true/false      | false   |           |                                  |
+| --httpTransportHost | Remote host of the HTTP logging endpoint                         | string  |                 | N/A     |           |                                  |
+| --httpTransportPort | Remote port of the HTTP logging endpoint                         | string  |                 | N/A     |           |                                  |
+| --httpTransportPath | Remote URI of the HTTP logging endpoint                          | string  |                 | N/A     |           |                                  |
+| --httpTransportSSL  | Use SSL for the HTTP logging endpoint                            | boolean | true/false      | false   |           |                                  |
+| --streamPath        | Stream transport path                                            | string  |                 | N/A     |           |                                  |
```

Comment:
Not sure if we need a --required column, as I would interpret that as meaning I have to do <command> --name <project-name>. Which you don't need to do given it's asked via the prompt

Reply:
Fair - I thought it might be worth mentioning in case of using this script with `--no-interactive` (as only `name` is required, not both of the interactive ones). But will remove if it's confusing.